chipsec/cfg/cfl.xml

```diff
@@ -66,29 +66,24 @@
     <register name="SBREG_BAR" type="pcicfg" bus="0" dev="0x1F" fun="1" offset="0x10" size="4" desc="Sideband Register Access BAR">
       <field name="RBA" bit="24" size="8" desc="Register Base Address"/>
     </register>
-
-    <register name="P2SBC" type="pcicfg" bus="0" dev="0x1F" fun="1" offset="0xE0" size="2" desc="P2SB Configuration Register">
-      <field name="HIDE" bit="8" size="1" desc="Hide SBREG_BAR"/>
-    </register>
-    <register name="P2SB_HIDE" type="pcicfg" bus="0" dev="0x1F" fun="1" offset="0xE1" size="1" desc="P2SB Configuration Register hide-unhide">
-      <field name="HIDE" bit="0" size="1" desc="Hide SBREG_BAR"/>
-    </register>
+    <!-- Since mm_msgbus needs SBREG_BAR we can't use mm_msgbus to get it...but it is at this position
+      <register name="SBREG_BAR" type="mm_msgbus" port="0xBC" offset="0x0900" size="4" desc="Sideband Register Access BAR">
+      <field name="RBA" bit="24" size="8" desc="Register Base Address"/>
+    </register>-->
 
     <!-- Power Management Controller -->
-    <register name="ABASE" type="pcicfg" bus="0" dev="0x1f" fun="2" offset="0x40" size="4" desc="ACPI Base Address">
-      <field name="STYPE" bit="0" size="1" desc="Space Type (always 1 - I/O space)"/>
-      <field name="BA"    bit="8" size="8" desc="Base Address"/>
-    </register>
     <register name="ACTL" type="pcicfg" bus="0" dev="0x1f" fun="2" offset="0x44" size="4" desc="ACPI Control">
       <field name="SCIS"    bit="0" size="2" desc="SCI IRQ Select"/>
       <field name="EN"      bit="7" size="1" desc="ACPI Enable"/>
       <field name="PWRM_EN" bit="8" size="1" desc="PWRM Enable"/>
     </register>
-    <register name="PWRMBASE" type="pcicfg" bus="0" dev="0x1f" fun="2" offset="0x48" size="4" desc="PM Base Address">
-      <field name="STYPE" bit="0"  size="1"  desc="Space Type (always 0 - memory space)"/>
-      <field name="BA"    bit="12" size="20" desc="Base Address"/>
+    <register name="PWRMBASE" type="mm_msgbus" port="0xBC" offset="0x0980" size="4" desc="PMC BAR0 PWRMBASE">
+      <field name="BA" bit="0" size="32" desc="PWRMBASE"/>
+    </register>
+    <register name="PMC_SHDW_BAR4" type="mm_msgbus" port="0xBC" offset="0x0990" size="4" desc="PMC BAR4 ABASE">
+      <field name="BA" bit="0" size="32" desc="ABASE"/>

```

Comment:
When attempting to access these registers on my system they only returned FFs.  I also do not find the registers in the public documentation.